### ISSUE TITLE
Feature request deck layout

### ISSUE THREAD

**Issue by sillen102** [leftwm/leftwm#137]
Hi.

I really, really like this project. I think you are doing an amazing job. I really like many of the features in LeftWM but there are a couple of things that at least I would need in order to be able to use this WM full time. One of those being the deck layout. It's a layout I use a lot in dwm.

This is how it works:
The screen is split in two between the master and the stack. When you open a new window it's placed on the stack, just like in lwm default layout. But in the deck layout the windows in the stack are placed on top of each other and not split. Like a deck of cards, hence the name. Basically it allows you to always have a two split no matter how many windows you have open and then you can switch focus between the windows in the stack, switch a window from the stack with the master etc.

It would also have been neat to implement the _NET_WM_STATE_HIDDEN flag for windows not in focus so that if you are using a terminal with transparency the windows behind it wouldn't be seen through if you set up opacity rules in picom.

Deck :
```
+--------------+-------------+
|              |             |
|              |             |
|              |             |
|       M      |      S1     |
|              |             |
|              |             |
|              |             |
+--------------+-------------+
```

**Comment by lex148** [comment]
I like this idea. Should be simple enough. I will see what I can do.

**Comment by grv07** [comment]
I am also waiting for deck layout feature.
Any update will be grt :)

**Comment by VuiMuich** [comment]
@lex148 are you working on this layout? If not, I might give this a shot, this looks like a mix of _main + stack_ and _monocle_.

(also might try to generalise the the focus fix for monocle)

**Comment by hertg** [comment]
@VuiMuich  
Was just browsing the open issues, i think this can be closed, as PR #269 was merged?

**Comment by VuiMuich** [comment]
Hey, thanks for this heads up.
I guess you are right, this issue should be closed by #269.
Since I can't close issues, as I am simply a normal contributor, we would nee to ping @sillen102 or @lex148 (theoretically @mautamu should be able to close issues as well, right?) to close this.
